File: sdk-examples/dotnet/AwsLambdaExample.cs
// Example: Invoking a Lambda function using AWS SDK for .NET

using Amazon.Lambda;
using Amazon.Lambda.Model;
using System;
using System.Text;
using System.Threading.Tasks;

class AwsLambdaExample
{
    static async Task Main()
    {
        var lambdaClient = new AmazonLambdaClient();

        var invokeRequest = new InvokeRequest
        {
            FunctionName = "your-lambda-function-name", // Replace with your Lambda function name
            Payload = "{\"key\":\"value\"}" // JSON payload
        };

        var response = await lambdaClient.InvokeAsync(invokeRequest);

        string responsePayload = Encoding.UTF8.GetString(response.Payload.ToArray());
        Console.WriteLine("Lambda response: " + responsePayload);
    }
}

File: sdk-examples/java/AwsLambdaExample.java
// Example: Invoking a Lambda function using AWS SDK for Java v2

import software.amazon.awssdk.services.lambda.LambdaClient;
import software.amazon.awssdk.services.lambda.model.InvokeRequest;
import software.amazon.awssdk.core.SdkBytes;

public class AwsLambdaExample {
    public static void main(String[] args) {
        LambdaClient lambda = LambdaClient.create();

        InvokeRequest invokeRequest = InvokeRequest.builder()
                .functionName("your-lambda-function-name") // Replace with your Lambda function name
                .payload(SdkBytes.fromUtf8String("{\"key\":\"value\"}")) // JSON payload
                .build();

        // Invoke the Lambda function
        var result = lambda.invoke(invokeRequest);

        String responsePayload = result.payload().asUtf8String();
        System.out.println("Lambda response: " + responsePayload);

        lambda.close();
    }
}